pkg/operator/workloads/api.go

```diff
@@ -253,10 +333,19 @@ func apiWorkloadSpecs(ctx *context.Context) ([]*WorkloadSpec, error) {
 			}
 		}
 
+		var spec metav1.Object
+
+		if api.ModelType == userconfig.TensorFlowModelType {
+			spec = tfAPISpec(ctx, api, workloadID, desiredReplicas)
+		}
+
+		if api.ModelType == userconfig.ONNXModelType {
+			spec = onnxAPISpec(ctx, api, workloadID, desiredReplicas)
+		}
```

Comment:
Nit: it might flow a little better if you remove the new lines on 337 and 341, and add a new line before 345